index.js

```diff
@@ -0,0 +1,121 @@
+// 1st exercise
+const greetings = function sayHello(name) {
```

Comment:
in function expression the name of function is a name of your variable. don't pass extra name after `function`.
Fix it here and in all places below